src/Pim/Component/Catalog/ProductValue/OptionsProductValue.php

```diff
@@ -73,6 +73,12 @@ public function getOptionCodes()
      */
     public function __toString()
     {
-        return implode(', ', $this->getOptionCodes());
```

Comment:
add a spec for that
you can take example on this PR https://github.com/jjanvier/pim-community-dev/pull/180/files#diff-29c8f7d5f2599ad6679a70892e30c4b2